Barley-break/js/main.js

```diff
@@ -0,0 +1,101 @@
+var game = document.getElementById("game");
```

Comment:
Global variable with a common name. It may override something or be overridden. Never ever use global variables